pynicotine/shares.py

```diff
@@ -127,7 +127,7 @@ def sendNumSharedFoldersFiles(self):
             shared_db = "sharedfiles"
 
         sharedfolders = len(conf["transfers"][shared_db])
-        sharedfiles = sum([len(x) for x in list(conf["transfers"][shared_db].values())])
+        sharedfiles = sum([len(x) for x in list(conf["transfers"][shared_db])])
```

Comment:
I'm not sure this does the right thing. It adds the length of the keys instead of the values, as in the old version. Unless you can explain why this is really the thing you want to do (I don't see it), this change should be reverted.